source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Continuous Integration with CircleCI
```

Comment:
How about "Configuring a GitHub Workflow with CircleCI for Drupal 8"?

@sukottokun would you like to weight in on this?